src/components/KCard/KCard.vue

```diff
@@ -1,244 +1,122 @@
 <template>
-  <section
-    :aria-describedby="contentId || undefined"
-    :aria-label="title ? title : undefined"
-    :aria-labelledby="!title && slots.title ? titleId : undefined"
-    class="kong-card"
-    :class="[borderVariant, {'hover': hasHover, 'kcard-shadow': hasShadow }]"
-  >
+  <section class="k-card">
     <div
-      v-if="showCardHead"
-      class="k-card-header"
-      :class="{ 'has-status': status || slots.statusHat }"
+      v-if="$slots.title || title || $slots.actions"
```

Comment:
suggestion: can you move this logic to a computed

Reply:
Done